util/changeToNew

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env sh
+
+#Script for updating version numbers and adding prerelease tags after creation of release branch. Arguments are the old and new version numbers as follows:
+(E.G. Release 1.25.0 to Pre-release 1.26.0)
```

Comment:
This seems like it probably should be preceded by a comment marker (`#`) as well?